File: Codeforces/10.cpp
#include <iostream>
#include <bits/stdc++.h>
#include <algorithm>
#include <sstream>
#include <string>
using namespace std;

void  check(int &n, string &s)
{
    int countA =0, countD=0;
    string sdf;
    for (int i=0; i<s.length(); i++){
        if (s[i]=='A'){
            countA++;
        }
        if (s[i]=='D'){
            countD++;
        }
    }
    if (countA==countD){
        cout << 'Friendship'<< endl;
    }
    if (countA>countD){
        cout << 'Anton'<<endl;
    }
    if (countA < countD)
    {
        cout << 'Danik'<<endl;
    }
}

int main()
{
    int n;
    string s;
    cin >> n;
    cin >> s;
    check(n, s);
}

File: competitive/Codeforces/10.py
n = int(input())
t = input()

ca = 0
cd = 0

for i in t:
    if i=="A":
        ca+=1
    if i=="D":
        cd+=1
if ca==cd:
    print("Friendship")
if ca>cd:
    print("Anton")
if ca<cd:
    print('Danik')
